client/components/wizard/docs/example.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Wizard from '../index';
+
+const First = () => <div style={ { textAlign: 'center' } }>This is the first step.</div>;
+const Second = () => <div style={ { textAlign: 'center' } }>This is the second step.</div>;
+const Third = () => <div style={ { textAlign: 'center' } }>This is the third step.</div>;
+const STEPS = {
+	FIRST: 'first',
+	SECOND: 'second',
+	THIRD: 'third',
+};
+const steps = [ STEPS.FIRST, STEPS.SECOND, STEPS.THIRD ];
+const components = {
+	[ STEPS.FIRST ]: <First />,
+	[ STEPS.SECOND ]: <Second />,
+	[ STEPS.THIRD ]: <Third />,
+};
+
+export default class WizardExample extends Component {
+	render() {
+		const { stepName } = this.props;
+
+		return (
+			<div>
+				<Wizard
+					basePath="/devdocs/design/wizard"
+					components={ components }
+					steps={ steps }
+					stepName={ stepName || steps[ 0 ] } />
```

Comment:
Nit -- I'd set the default at declaration level (or even via `defaultProps`, but see https://github.com/Automattic/wp-calypso/pull/16792/files#r132172906). Rationale is roughly that it makes sense for `stepName` to always be initialized.